test/performance/load-test/main.go

```diff
@@ -161,32 +166,27 @@ func main() {
 	}
 
 	// Get the Kubernetes version from the API server.
-	version, err := kubeclient.Get(ctx).Discovery().ServerVersion()
+	c := kubeclient.Get(ctx)
+	if c == nil {
+		log.Fatalf("Failed to fetch %T from context.", (kubernetes.Interface)(nil))
+	}
+
+	version, err := c.Discovery().ServerVersion()
 	if err != nil {
 		log.Fatalf("Failed to fetch kubernetes version: %v", err)
 	}
+	// '.' is an invalid char in mako tags. Replace with "_"
+	tags = append(tags, fmt.Sprintf("kubernetes=%s", strings.ReplaceAll(version.String(), ".", "_")))
```

Comment:
feels like all of this should be in `mako.Setup()` and only TBC should be an extra tag.

Reply:
we need to inject kubeclient with informers into the context, which is not needed in dataplane-probe. Do we want to do that for all benchmarks?

Reply:
We want the K8s tag on everything, so we want the client everywhere.  We can follow up.